src/components-examples/journey-maps/angular/journey-maps-basic/journey-maps-basic-example.ts

```diff
@@ -1,4 +1,6 @@
 import { Component } from '@angular/core';
+import { SbbNotificationModule } from '@sbb-esta/angular/notification';
+import { SbbJourneyMapsModule } from '@sbb-esta/journey-maps/angular/journey-maps.module';
```

Comment:
```suggestion
import { SbbJourneyMapsModule } from '@sbb-esta/journey-maps';
```
@mstankala, after applying this change, you need to run `yarn baz` and commit all changes. Stackblitz will work but the map is not showing due to the missing token. Do you already have an idea how to solve this?

Reply:
@mhaertwig No, it still doesn't work.

Reply:
OK, with your suggested change, it work now. Thx 
@mhaertwig What do you suggest, how can we pass the test api-key into the Stackblitz example?

Reply:
Hello @mstankala 
Look for `${sbbLean}` in the code for an example on how to pass data into StackBlitz

Reply:
How can I pass all file with exmaple-data from 'src/components-examples/journey-maps/angular/journey-maps-full/mock-response/*' into StackBlitz. 
What is the best way?

Reply:
I'm not fond of it, as this can very easily break when changes are implemented.
But I guess I can live with it.